test/repro/reg_test_5363_admin_monitor_caching_sha2.bash

```
@@ -0,0 +1,228 @@
+
+restore() {
+	setvar 'admin-admin_credentials' "$BASE_CREDS" 'LOAD ADMIN VARIABLES TO RUNTIME;'
+	setvar 'mysql-default_authentication_plugin' 'mysql_native_password' 'LOAD MYSQL VARIABLES TO RUNTIME;'
+	setvar 'mysql-monitor_password' 'monitor' 'LOAD MYSQL VARIABLES TO RUNTIME;'
+}
+trap restore EXIT
+
+# ----------------------------------------------------------- preconditions ---
+hdr "Preconditions"
+[ "$(adm 'SELECT count(*) FROM runtime_mysql_users;')" = "0" ] \
+	&& ok "no mysql_users rows (nothing can shadow the monitor credential)" \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)